behave/features/steps/specialist_provider.py

```diff
@@ -434,3 +441,16 @@ def step_impl(context):
     # Find the same date as the uploaded csv file
     table_row = context.helperfunc.find_by_xpath(f"//table/tbody/tr/td[text()='{new_date} Upload']")
     assert table_row is not None
+
+
+@step(u'I am given details of the errors in each line of the csv file')
+def step_impl(context):
+    context.csv_page = context.helperfunc.find_by_xpath("//*[@id='wrapper']")
+    # Check to make sure there are list items in the HTML unordered list attribute
+    xpath = "//ul[contains(@class, 'ErrorSummary-list')]/li"
+    error_list = len(context.helperfunc.driver().find_elements_by_xpath(xpath))
+    # There are three rows that error in the invalid CSV, confirm they are visible.
+    assert error_list == 3
+    error_message = "Row: 1 - The Fixed Fee code you have entered is not valid with time spent on the case"
+    # Find one of the error messages displayed, confirm its error message.
+    context.csv_page.find_element_by_xpath(f"//li[text()='{error_message}']")
```

Comment:
should there be another assert here or are you happy that it just fails if there is no such message?